civicrm_entity.info.yml

```diff
@@ -6,4 +6,5 @@ core_version_requirement: ^8.8 || ^9
 dependencies:
   - civicrm
   - drupal:datetime
+  - drupal:filter
```

Comment:
We call functions for this module. Generally, it should always be available. but this is a safeguard